packages/video_player/video_player/example/integration_test/video_player_test.dart

```diff
@@ -56,7 +59,7 @@ void main() {
       (WidgetTester tester) async {
         final VideoPlayerController networkController =
             VideoPlayerController.network(
-          'https://cph-p2p-msl.akamaized.net/hls/live/2000341/test/master.m3u8',
+          'https://flutter.github.io/assets-for-api-docs/assets/videos/hls/bee.m3u8',
```

Comment:
How do you upload stuff to assets-for-api-docs? Is it the website repo?

Reply:
It's a separate repo; this was: https://github.com/flutter/assets-for-api-docs/pull/178